ufl/finiteelement/tensorproductelement.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, *elements, **kwargs):
     def __repr__(self):
         return "TensorProductElement(" + ", ".join(
             repr(e) for e in self._sub_elements
-        ) + f", {repr(self._cell)})"
+        ) + f", cell={repr(self._cell)})"
```

Comment:
Could you move the three lines into one? It's very hard to read being split across three lines. I think all on one line will pass the linter.

Reply:
Sure. Done!